docs/tutorials/create-transactions.md

```diff
@@ -181,29 +181,45 @@ const tx = nearAPI.transactions.createTransaction(
 ___
 
 ## Sign Transaction
-Now that the transaction is created, we sign it before sending it to the NEAR blockchain.
-  - Using [`nearAPI`](/docs/tutorials/create-transactions#imports), we call on `signTransaction()` to perform this task.
-  - This method takes:
-    - `tx` _([transaction](/docs/tutorials/create-transactions#constructing-the-transaction) we just created)_
-    - `signerObject` _(this is nested in the `near` object we created earlier with [`setupNear()`](/docs/tutorials/create-transactions#setting-up-connection-to-near))_
-    - `senderID` _(account ID of the signer / sender passed as a string)_
-    - `networkID` _(network we want to connect to (`testnet` is default))_
+Now that the transaction is created, we sign it before sending it to the NEAR blockchain. At the lowest level, there are four steps to this process. 
+
+1) Using [`nearAPI`](/docs/tutorials/create-transactions#imports), we call on `serialize()` to serialize the transaction in [BORSH](https://borsh.io/).
 ```js
-const [txHash, signedTx] = await nearAPI.transactions.signTransaction(
-  tx, 
-  near.connection.signer, 
-  sender, 
-  networkId
-  );
+ const serializedTx = nearAPI.utils.serialize.serialize(
+    nearAPI.transactions.SCHEMA, 
+    transaction
+    );
 ```
 
+2) Hash the transaction using a `sha256` cryptographic hashing algorithm.
+```js
+const hashedTx = new Uint8Array(sha256.sha256.array(serializedTx));
```

Comment:
I think `hashedTx` is a bit misleading as the result is a hash of the serialized transaction blob (`serializedTxHash`)